code/messaging/src/main/java/com/adobe/marketing/mobile/messaging/internal/AJOPayloadHandler.java

```diff
@@ -139,7 +157,7 @@ void fetchMessages() {
         messagesRequestEventId = event.getUniqueIdentifier();
 
         // send event
-        Log.debug(LOG_TAG, SELF_TAG, "Dispatching edge event to fetch in-app messages.");
+        Log.debug(LOG_TAG, SELF_TAG, "Dispatching edge event to retrieve AJO message payloads.");
```

Comment:
Suggest rephrase: Dispatching an edge event to retrieve in-app or feed message definitions.